nixos/modules/programs/x2goserver.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.x2goserver;
+in {
+  options.programs.x2goserver.enable = mkEnableOption "x2goserver";
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.x2goserver ];
+    users.extraGroups.x2gouser = {};
+    users.extraUsers.x2gouser = {
+      createHome = true;
+      home = "/var/lib/x2go";
+      group = "x2gouser";
+    };
+    users.extraGroups.x2goprint = {};
+
+    security.wrappers.x2gosqliteWrapper = {
```

Comment:
Reading upstream I saw mention to adding rules to the `sudoers` file. Is this related at all? Do we need these rules in `sudoers`?

Reply:
This isn't related, but I did add a sudoers change below, with a link to the original bug-report (it sets some env-var necessary for sudoing GUI apps sometimes).